src/network/networkd-route.c

```diff
@@ -1201,3 +1201,53 @@ int config_parse_route_mtu(
         TAKE_PTR(n);
         return 0;
 }
+
+int config_parse_ipv4ll_route(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Network *network = userdata;
+        _cleanup_(route_freep) Route *n = NULL;
+        int r;
+
+        assert(filename);
+        assert(section);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = parse_boolean(rvalue);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r,
+                           "Could not parse %s=\"%s\", ignoring assignment: %m", lvalue, rvalue);
+                return 0;
+        }
+        if (r == 0)
+                return 0;
```

Comment:
hmm, but this is weird, no? it means if i have a .network file with:

```
[Network]
IPv4LLRoute=yes
IPv4LLRoute=no
```

then I end up with the route in place. But our usual rule is that the last setting wins...

And this would result in a duplicate route:

```
[Network]
IPv4LLRoute=yes
IPv4LLRoute=yes
```

no?

Hence maybe keep a ref to this special route in the Network structure so that we can remove it again or check if it already exists. or maybe just create the route after a network object was fully loaded